qiskit/pulse/instructions/set_frequency.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Set channel frequency.
```

Comment:
I've been trying to avoid module descriptions that just repeat the file name. you can check out my open PRs to terra for what I mean. I also don't mind updating this myself when later I update the API ref docs to point to instructions rather than commands